src/tests/databases/tables.sql

```diff
@@ -0,0 +1,195 @@
+);
+create index assembly_sequence_assembly_id_2a84ddcb on assembly_sequence (assembly_id);
+create table attribute
+(
+    attribute_id int
+        primary key,
+    name         TEXT                  not null,
+    label        TEXT                  not null,
+    description  TEXT                  null,
+    type         TEXT default 'string' null,
+    constraint name unique (name),
+    constraint name_2 unique (name),
+    constraint name_3 unique (name)
+);
+create table dataset_source
+(
+    dataset_source_id int primary key,
+    type              TEXT not null,
+    name              TEXT not null,
+    constraint name unique (name)
+);
+create table dataset_type
+(
+    dataset_type_id int primary key,
+    name            TEXT not null,
+    label           TEXT not null,
+    topic           TEXT not null,
+    description     TEXT null,
+    details_uri     TEXT null
+);
+create table dataset
+(
+    dataset_id        int primary key,
+    dataset_uuid      TEXT                     not null,
+    dataset_type_id   int                      not null,
+    name              TEXT                     not null,
+    version           TEXT                     null,
+    created           datetime(6)              not null,
+    dataset_source_id int                      not null,
+    label             TEXT                     not null,
+    status            TEXT default 'Submitted' null,
+    constraint dataset_uuid unique (dataset_uuid),
+    constraint dataset_dataset_source_id_fd96f115_fk_dataset_s foreign key (dataset_source_id) references dataset_source (dataset_source_id),
+    constraint dataset_dataset_type_id_47284562_fk_dataset_type_dataset_type_id foreign key (dataset_type_id) references dataset_type (dataset_type_id)
+);
+create index dataset_type_id_eb55ae9a
+    on dataset (dataset_type_id);
+create table dataset_attribute
+(
+    dataset_attribute_id int primary key,
+    value                TEXT not null,
+    attribute_id         int  not null,
+    dataset_id           int  not null,
+    constraint dataset_attribute_dataset_id_attribute_id__d3b34d8c_uniq unique (dataset_id, attribute_id, value),
+    constraint dataset_attribute_attribute_id_55c51407_fk_attribute foreign key (attribute_id) references attribute (attribute_id),
+    constraint dataset_attribute_dataset_id_2e2afe19_fk_dataset_dataset_id foreign key (dataset_id) references dataset (dataset_id)
+);
+create index dataset_attribute_dataset_id_2e2afe19 on dataset_attribute (dataset_id);
+create table django_migrations
+(
+    id      int primary key,
+    app     TEXT        not null,
+    name    TEXT        not null,
+    applied datetime(6) not null
+);
+create table ensembl_site
+(
+    site_id int primary key,
+    name    TEXT not null,
+    label   TEXT not null,
+    uri     TEXT not null
+);
+create table ensembl_release
+(
+    release_id   int primary key,
+    version      decimal(10, 1) not null,
```

Comment:
> FYI: This comment is for @marcoooo and @vinay-ebi, just to keep this in mind.

In the newer Python versions (e.g: 3.10) I ran into this error
```
TypeError: Cannot set ensembl_metadata.Release.release_version to Decimal('108.0'): Decimal('108.0') has type <class 'decimal.Decimal'>, but expected one of: (<class 'float'>, <class 'numbers.Real'>) for field Release.release_version
```
Whenever `release_version` is used, this can be fixed by changing `release_version` data type from `decimal` to `float` in the DB (might be related to [this](https://github.com/protocolbuffers/protobuf/issues/3913#issuecomment-1065436793)).

Reply:
This is something to keep in mind, yes. Although current version used in docker container is 3.8.